conan/api/model/remote.py

```diff
@@ -2,16 +2,24 @@
 
 
 class Remote:
-
+    """
+    The Remote class represents a remote registry of packages. It's a read-only opaque object that
+    should not be created directly, but obtained from the relevant ``RemotesAPI`` subapi methods.
+    """
     def __init__(self, name, url, verify_ssl=True, disabled=False, allowed_packages=None,
                  remote_type=None):
         self.name = name  # Read only, is the key
+        """User-facing remote name"""
         self.url = url
+        """URL endpoint of the Conan remote"""
         self.verify_ssl = verify_ssl
+        #"""Whether to verify SSL certificates"""
         self.disabled = disabled
+        #"""If this remote is disabled, it won't be taken into account for Conan operations"""
         self.allowed_packages = allowed_packages
+        #"""Which packages this remote is allowed to fetch, or None if all packages are allowed"""
```

Comment:
This are right now not documented, left this here to discuss if we want any of these attributes to be

Reply:
Maybe better to make read-only properties explicitly, as changing them on the fly might be problematic?
If we are exposing this object, maybe start slowly and considering the APIs more carefully, maybe lets start with just the class name, as this would require some thinking?

Reply:
I documented `name` and `url` at first because `auth_remote.py` documentation uses `url` in its usage example, but we can probably live with that until we better decide what gets exposed 👍